BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+ExchangeRateService service = new ExchangeRateService();
```

Comment:
Delete this line.  We need to introduce our service with Dependency Injection. So it will be easy to manage lifetime of service.

Write this instead :
`builder.Services.AddSingleton<IExchangeRateService, ExchangeRateService>();`